kolibri/plugins/learn/assets/src/views/index.vue

```diff
@@ -168,6 +207,11 @@
         return { name: PageNames.EXAM_LIST, params: { channel_id: this.channelId } };
       },
     },
+    watch: {
+      searchTerm(val) {
```

Comment:
If it was a computed prop based on `this.searchTerm` why wouldn't `this.searchQuery` recompute when it changed?